MAINTAINERS.md

```diff
@@ -0,0 +1,5 @@
+# Maintainers
+
+This repository is maintained by the Philips IGT Cloud Team.
+
+Koen de Laat <koen.de.laat@philips.com>
```

Comment:
Since this is open source: I would use `@koendelaat` instead of your philips address